admin_guide/node_problem_detector.adoc

```diff
@@ -0,0 +1,428 @@
+          readOnly: true
+        - name: config
+          mountPath: /config
+          readOnly: true
+      volumes:
+      - name: log
+        # Config `log` to your system log directory
+        hostPath:
+          path: /var/log/ <3>
+      - name: kmsg
+        hostPath:
+          path: /dev/kmsg
+      - name: localtime
+        hostPath:
+          path: /etc/localtime
+      - name: config
+        configMap:
+          name: node-problem-detector-config <4>
+----
++
+<1> Node problem detector binding address and port.
+<2> List of paths to link:https://github.com/kubernetes/node-problem-detector/blob/master/config/kernel-monitor.json[system log monitor configuration files], 
+comma separated. Node problem detector will start a separate log monitor for each configuration.
+<3> Path to your system log directory.
+<4> Configuration map used by the node problem detector. You can xref:admin-guide-node-problem-detector-overwrite[use your own configuration file], as needed.
+
+. Edit the file for your environment. 
+
+. Save the file as any name, such as *_node-problem-detector.yaml_*.
+
+. Run the following command to start node problem detector: 
++
+[source,bash]
+----
+oc create -f <your_yaml_file>.yaml
+----
+////
+
+[[admin-guide-node-problem-detector-customize]]
+== Configuring Custom Logging and Monitoring Tools
+
+You can use xref:admin-guide-node-problem-detector-fbk[file-based kernel logging], or 
+xref:admin-guide-node-problem-detector-monitor[custom monitoring tools]. 
+
+To enable plugins, edit the node problem detector daemonset:
+
+. Edit the Node Problem Detector configuration map with a text editor.
+[source,bash]
++
+----
+oc edit configmap -n <project> <node-problem-detector-name>
+----
++
+For example:
++
+----
+oc edit configmap -n openshift-node-problem-detector node-problem-detector
+----
+
+. To display node problem detector output to standard output (stdout) and standard error (stderr) edit the node problem detector daemonset
++
+[source,yaml]
+----
+spec:
+  containers:
+  - command:
+    - node-problem-detector
+    - --alsologtostderr=true <1>
+      --log_dir="/tmp" <2>
+
+----
+
+<1> Sends the output to standard output (stdout).
+<2> Path to the error log.
++
+The node problem detector starts a separate log monitor for each configuration. You can use different log monitors to monitor different system logs.
+
+. Add the following flag to set the output and enable kernel-monitor, and docker-monitor.jason:
++
+----
+spec:
+  containers:
+  - command:
+    - node-problem-detector
+    - --system-log-monitors=/etc/npd/kernel-monitor.json,/etc/npd/docker-monitor.json <1>
+----
++
+<1> Comma-separated path to the plugin configuration files.
+
+[[admin-guide-node-problem-detector-uninstall]]
+== Uninstall the Node Problem Detector
+
+To uninstall the node problem detector:
+
+. Add following options in Ansible inventory file:
++
+[source,yaml]
+----
+[OSEv3:vars]
+openshift_node_problem_detector_state=absent
+----
```

Comment:
@joelsmith Should this be openshift_node_problem_detector_install=**false**

Reply:
No, it's correct as written.